index.js

```diff
@@ -70,7 +70,7 @@ function create(win, opts) {
 			menuTpl.push(...opts.append(props));
 		}
 
-		if (isDev) {
+		if (opts.showInspectElement || isDev) {
```

Comment:
You can't do this as if set to `false` it would still auto-detect. Setting to `false` should disable it no matter what and `true` should enable it no matter what.